src/Servers/Kestrel/Transport.Quic/README.md

```diff
@@ -1,3 +1,167 @@
-For external contributors, msquic.dll isn't available publicly yet. See https://github.com/aspnet/Announcements/issues/393.
+# Getting start with HTTP/3 on Windows and Linux
 
-Credit to Diwakar Mantha and the Kaizala team for the MsQuic interop code.
\ No newline at end of file
+This is a getting started guide for using HTTP/3 on Windows and Linux.
+
+## Prerequisites
+
+1. Download the latest 5.0-preview build of .NET from <https://dotnet.microsoft.com/download/dotnet/5.0.>
+2. Download Edge Canary <https://www.microsoftedgeinsider.com/en-us/download.> This is required to browse websites with HTTP/3.
+
+### Windows
+
+1. Latest [Windows Insider Builds](https://insider.windows.com/en-us/), Insiders Fast build. This is required for Schannel support for QUIC.
+2. Enabling TLS 1.3. Add the following registry keys to enable TLS 1.3.
+
+    ```text
+    reg add "HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\SecurityProviders\SCHANNEL\Protocols\TLS 1.3\Server" /v DisabledByDefault /t REG_DWORD /d 1 /f
+    reg add "HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\SecurityProviders\SCHANNEL\Protocols\TLS 1.3\Server" /v Enabled /t REG_DWORD /d 1 /f
+    reg add "HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\SecurityProviders\SCHANNEL\Protocols\TLS 1.3\Client" /v DisabledByDefault /t REG_DWORD /d 1 /f
+    reg add "HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\SecurityProviders\SCHANNEL\Protocols\TLS 1.3\Client" /v Enabled /t REG_DWORD /d 1 /f
+    ```
```

Comment:
I'm assuming these are global reg keys. We should add a warning that it can cause unintended side effects for your existing applications.